egs2/chime7_task1/asr1/local/gen_task1_data.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -euo pipefail
+function contains ()  { [[ $1 =~ (^|[[:space:]])"$2"($|[[:space:]]) ]]; }
+
+stage=1
+cmd=run.pl
+skip_stages="-1"
+# this script creates and prepares the official Task 1 data from chime6, dipco and mixer6 datasets.
+chime7_root=
+chime5_root= # chime5 needs to be downloaded manually, but you can skip it if you have already CHiME-6
+chime6_root= # will be created automatically from chime5
+# but if you have it already you can use your existing one.
+dipco_root=
+mixer6_root=
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./utils/parse_options.sh || exit 1
+
+
+if ! [ -d chime5_root ]; then
```

Comment:
`$chime5_root` instead of just `chime5_root`?

Reply:
This was noted also from NVIDIA https://github.com/espnet/espnet/pull/4934 fixes it